lib/auth0/api/v2/actions.rb

```diff
@@ -16,10 +16,7 @@ module Actions
         # @param page [integer] The page number. Zero based.
         # @param installed [boolean] When true, return only installed actions. When false, return only custom actions. Returns all actions by default.
         # @return [json] Actions and pagination info
-        def actions(trigger_id, action_name, deployed: nil, per_page: nil, page: nil, installed: nil)
-          raise Auth0::MissingTriggerId, 'Must supply a valid trigger_id' if trigger_id.to_s.empty?
-          raise Auth0::MissingActionName, 'Must supply a valid action_name' if action_name.to_s.empty?
-
+        def actions(trigger_id: nil, action_name: nil, deployed: nil, per_page: nil, page: nil, installed: nil)
           request_params = {
             trigger_id: trigger_id,
             action_name: action_name,
```

Comment:
Unfortunately the original implementation looks broken here, as these should be (according to [the API](https://auth0.com/docs/api/management/beta/v2/get-actions)):

```ruby
triggerId: trigger_id,
actionName: action_name,
```

I'm curious though, did you test this against the API? As I can't get your implementation to work, I get the `"Query validation error: 'Additional properties not allowed: trigger_id'."` error with the PR as it is now.

Reply:
Yeah I can see the same. If they are passed, however, we get an error.

Reply:
I've provided a fix in https://github.com/auth0/ruby-auth0/pull/478/commits/cb1ade0bdc1458e2e60f59a0bdb12119592ca980 that makes these arguments optional positional for backwards compatibility.